docs/developingFirstMod.rst

```diff
@@ -0,0 +1,159 @@
+
+.. _physicalItem:
+Creating the PhysicalItem
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Create a blueprint class with name ``ExampleItem_BP`` and in the creation dialogue open All classes.
+
+Select PhysicalItem to inherit from, create and open the class.
+
+First of all we want to set the model for that item so that we have something to look at.
+
+For that we will need to :ref:`import the mesh <importingMesh>`
+
+.. _importingMesh:
+Importing the mesh
+^^^^^^^^^^^^^^^^^^^^^^
+
+    1. Open your mod folder in the **Content Browser**
+    2. Download this example mesh :download:`exampleMesh.fbx`
+    3. Drag&drop your mesh into the **Content Browser** and click ``Import all``
+    4. Open the mesh and in the **Details** tab search for ``Has navigation Data`` and uncheck that checkmark.
+
+.. warning::
+    If you don't uncheck ``Has navigation Data`` the game will crash when loading your mod.
+
+After importing the mesh, open the item you created and click on **StaticMeshComponent**.
+
+.. note:: 
+    If you don't see the **StaticMeshComponent** click on **Open Full Blueprint Editor**
+
+There in the **Details** tab set your mesh in ``Static Mesh`` field.
+
+Click on **Compile** and then **Save**.
+
+.. _itemType:
+Creating the ItemType
+~~~~~~~~~~~~~~~~~~~
+
+Now that we have created the PhysicalItem it's time to make the ItemType for it.
+
+Create a blueprint class with name ``ExampleItem_IT``, and in the creation dialogue open All classes.
+
+Select ItemType to inherit from, create and open the class.
+
+This is a simple item so we will leave most of the options untouched, you can experiment with them yourself and see what they do.
+
+* Set **Pickup Actor** to be ``TestItem_BP``.
+* Open **Construction Recipe**
+    * Press "+" on the **Ingredients** field.
+    * Set Item type of the ingredient to **Astronium**
+    * Set Count of the ingredient to 1.
+* Set **Catalog Data** to ``Item Catalog Data``
+
+- ``Base Item Type`` in **Catalog Data** is used to determine where in the item catalog the item will be listed.
+- ``Varitation Sequence Number`` in **Catalog Data** is used to determine the order in which the item will be listed in the item catalog.
+- ``Catalog Mesh`` in **Catalog Data** is used to determine the mesh that will be displayed in the item catalog.
+
+Set ``Base Item Type`` to ``Consumable_JumpJet_IT`` so it gets listed near jetpacks and hoverboards.
+
+Set ``Catalog Mesh`` to the mesh we imported earlier.
+
+
+Open **Control Symbol** section and fill the fields out like this:
+
+* **Name**: ``TestItem``
+* **All caps Name**: ``TESTITEM``
+* **Tooltip Subtitle**: ``Test Item``
+* **Description**: ``This is a test item.``
+
+.. _cookingMod:
+Cooking the mod
+================
+
+Click on **File** > **Cook Content for Windows**
+
+While the content is being cooked create a folder in file explorer with the name ``000-TutorialMod-0.1.0_P`` and open it.
+
+Inside this folder create a file called ``metadata.json``
+
+Fill this file out like this
```

Comment:
Maybe also add a comment here to replace YOUR_NAME. I feel like a lot of people are just going to copy paste everything without reading the actual json.